bigbone/src/main/kotlin/social/bigbone/api/entity/Poll.kt

```diff
@@ -0,0 +1,86 @@
+package social.bigbone.api.entity
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Represents a poll attached to a status.
```

Comment:
Please add a link to the official website entity description (please also check the other new entities added). Thanks 🙂